### PR TITLE
feat: encourage using random treasury id

### PR DESCRIPTION
**Pull request by guibescos**
To avoid treasury account becoming hot

### CODE REVIEW

File: target_chains/solana/cli/src/main.rs [hunk 1]
```diff
@@ -186,16 +185,9 @@ fn main() -> Result<()> {
                     },
                 );
 
-            // We need to send some rent to the treasury account, otherwise it won't be able to accept incoming transfers
-            let pay_treasury_rent = system_instruction::transfer(
```
Comment (guibescos) [inline target_chains/solana/cli/src/main.rs:190]: drive-by cleanup

**Review by jayantk — APPROVED** [review]
